qiskit/visualization/circuit/qcstyle.py

```diff
@@ -21,146 +21,143 @@
-                'ry': 'R_Y',
-                'rz': 'R_Z',
-                'rxx': 'R_{XX}',
-                'ryy': 'R_{YY}',
-                'rzx': 'R_{ZX}',
-                'rzz': 'ZZ',
-                'reset': '\\left|0\\right\\rangle',
-                'initialize': '|\\psi\\rangle'
-            }
-
-    displaycolor (dict): the color codes to use for each circuit element in
-        the form (gate_color, text_color). Colors can also be entered without
-        the text color, such as 'u1': '#FA74A6', in which case the text color
-        will always be `gatetextcolor`. The `displaycolor` dict can contain
-        any number of elements. User names and labels may be used as keys,
-        which allows for custom colors for user-created gates. The default
-        values are (`default.json`)::
-
-            {
-                'u1': ('#FA74A6', '#000000'),
-                'u2': ('#FA74A6', '#000000'),
-                'u3': ('#FA74A6', '#000000'),
-                'id': ('#05BAB6', '#000000'),
-                'u': ('#BB8BFF', '#000000'),
-                'p': ('#BB8BFF', '#000000'),
-                'x': ('#05BAB6', '#000000'),
-                'y': ('#05BAB6', '#000000'),
-                'z': ('#05BAB6', '#000000'),
-                'h': ('#6FA4FF', '#000000'),
-                'cx': ('#6FA4FF', '#000000'),
-                'ccx': ('#BB8BFF', '#000000'),
-                'mcx': ('#BB8BFF', '#000000'),
-                'mcx_gray': ('#BB8BFF', '#000000'),
-                'cy': ('#6FA4FF', '#000000'),
-                'cz': ('#6FA4FF', '#000000'),
-                'swap': ('#6FA4FF', '#000000'),
-                'cswap': ('#BB8BFF', '#000000'),
-                'ccswap': ('#BB8BFF', '#000000'),
-                'dcx': ('#6FA4FF', '#000000'),
-                'cdcx': ('#BB8BFF', '#000000'),
-                'ccdcx': ('#BB8BFF', '#000000'),
-                'iswap': ('#6FA4FF', '#000000'),
-                's': ('#6FA4FF', '#000000'),
-                'sdg': ('#6FA4FF', '#000000'),
-                't': ('#BB8BFF', '#000000'),
-                'tdg': ('#BB8BFF', '#000000'),
-                'sx': ('#6FA4FF', '#000000'),
-                'sxdg': ('#6FA4FF', '#000000')
-                'r': ('#BB8BFF', '#000000'),
-                'rx': ('#BB8BFF', '#000000'),
-                'ry': ('#BB8BFF', '#000000'),
-                'rz': ('#BB8BFF', '#000000'),
-                'rxx': ('#BB8BFF', '#000000'),
-                'ryy': ('#BB8BFF', '#000000'),
-                'rzx': ('#BB8BFF', '#000000'),
-                'reset': ('#000000', '#FFFFFF'),
-                'target': ('#FFFFFF', '#FFFFFF'),
-                'measure': ('#000000', '#FFFFFF'),
-            }
+    """Creates a Default Style dictionary that contains numerous options that
+    define the style of the output circuit visualization. The style dict is used
+    by the mpl or latex output.
+
+    Args:
+        **name (str)**: the name of the style. The name can be set to ``iqp``,
```

Comment:
To remain consistent with the code base, could you remove the bold face and just put `<argument>: <description>` instead of `**<argument>**: <description>` here?

Reply:
Sure! I only did this because @Eric-Arellano said to do so in issue #11037.

Reply:
Ah! I think they may have been talking about the appearance in bold face, not that we have to wrap it in `**`. The boldface should appear automatically for arguments 🙂